web/console-extensions.json

```diff
@@ -32,7 +32,6 @@
       "id": "alerting",
       "name": "%plugin__monitoring-plugin~Alerting%",
       "href": "/monitoring/alerts",
-      "perspective": "admin",
```

Comment:
The change needed here isn't the removal of the perspective but rather the addition of virtualization perspective specific links. While the perspective is optional, when it isn't present, it is presumed to be the default perspective ([link](https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md#consolenavigationhref)) which is the admin perspective.

The link `/monitoring/alerts` should refer to a location in a single perspective. For the developer perspective there is `/dev-monitoring/ns/:ns/alerts`, and for the virtualization perspective there should be a third `/virt-monitoring/alerts` URL.

These new links should then be included in the various usePerspective functions so that all of the pages link together correctly.